content/en/serverless/installation/python.md

```diff
@@ -0,0 +1,187 @@
+4. Select the option to *Provide a layer version ARN*.
+5. Enter the Datadog Lambda Library ARN from the table above.
+6. Navigate to the Environment Variables section of your function and add a new `“DD_FLUSH_TO_LOG”` variable set to `“true”`.
+
+These steps need to be repeated for every function you wish to trace.
+
+### Instrument your Code
+
+Instrument your functions to ingest traces into Datadog:
+
+```
+from datadog_lambda.wrapper import datadog_lambda_wrapper
+from ddtrace import tracer
+
+@datadog_lambda_wrapper
+def hello(event, context):
+return {
+    "statusCode": 200,
+    "body": get_message()
+}
+
+@tracer.wrap()
+def get_message():
+return "Hello from serverless!"
+```
+
+### Subscribe the Forwarder to Log Groups
+
+You need the Datadog Forwarder to subscribe to each of your function’s log groups to send traces and enhanced metrics to Datadog.
+
+You can quickly verify that you’ve installed the Datadog Forwarder on the [AWS Forwarder page][2]. If you have not yet installed the Forwarder, follow the instructions to install the Forwarder in the same AWS region as the Lambda functions you are monitoring.
+
+1. To start, navigate to your AWS Dashboard for the Datadog Forwarder. Then, manually add a function trigger.
+2. Configure the trigger to be linked to your function’s CloudWatch Log Group, add a filter name (but feel free to leave the filter empty) and add the trigger:
+
+The Datadog Forwarder will now send enhanced metrics and traces from your function to Datadog.
+
+
+[1]: https://github.com/DataDog/datadog-lambda-layer-python/releases
+[2]: https://console.aws.amazon.com/cloudformation/home#/stacks?filteringText=forwarder
+{{% /tab %}}
+{{% tab "Other" %}}
+
+### Install the Datadog Lambda Library
+
+The Datadog Lambda Library can be installed from PyPI by running the following command
+or adding datadog-lambda to your project's requirements.txt:
+
+```
+pip install datadog-lambda
+```
+
+Then, using the AWS Console or the AWS CLI, add a new `“DD_FLUSH_TO_LOG”` environment variable set to `“true”`. This step needs to be repeated for every function you wish to trace.
+
+### Instrument your Code
+
+Instrument your functions to ingest traces into Datadog:
+
+```
+from datadog_lambda.wrapper import datadog_lambda_wrapper
+from ddtrace import tracer
+
+@datadog_lambda_wrapper
+def hello(event, context):
+return {
+    "statusCode": 200,
+    "body": get_message()
+}
+
+@tracer.wrap()
+def get_message():
+return "Hello from serverless!"
+```
+
+### Subscribe the Forwarder to Log Groups
+
+You need the Datadog Forwarder to subscribe to each of your function’s log groups to send traces and enhanced metrics to Datadog.
+
+You can quickly verify that you’ve installed the Datadog Forwarder here. For more information on the Forwarder ARN, or to install the forwarder see the [official CloudFormation documentation][1]. Make sure the Datadog Forwarder is in the same AWS region as the Lambda functions you are monitoring.
```

Comment:
We should probably link this at "here": https://console.aws.amazon.com/cloudformation/home#/stacks?filteringText=forwarder

Reply:
This will probably go for Node + Ruby as well